### ISSUE TITLE
Chyby na webu

### ISSUE THREAD

**Issue by kalich5** [Yendy/petkari.cz#40]
Našel jsem chybu která je skrz celý web. Kde je "Přidat komentář" tak pod ním je "Vaše jménoKalich". chyby tam mezera před přezdívkou. Možná bych to udělal "Vaše jméno: Kalich", co myslíte? Budu dál hledat, když by něco napíšu

**Comment by kalich5** [comment]
U klubovny je období provozu. Chybí tam možnost vyšrtnout možnost "do" a tím pádem se nedá u klubovny V Tůních zadat že je to aktuální místo. Tohle musíme nějak vyřešit.

**Comment by kalich5** [comment]
Nejde zadat autor u fotek, vypíše to chybu

**Comment by kalich5** [comment]
![chyba](https://cloud.githubusercontent.com/assets/8988898/15504437/4a1f321e-21bf-11e6-9f95-065a57d350aa.jpg)

**Comment by kalich5** [comment]
![chyba](https://cloud.githubusercontent.com/assets/8988898/15823250/389f9886-2bfa-11e6-951a-1a92c3e10004.png)

**Comment by Yendy** [comment]
Chyba AJAX HTTP se mi už také párkrát objevila. Obvykle pomohlo odklepnout to a zadat řádek znovu. Pak už to běželo. Občas bylo třeba to udělat dvakrát nebo třikrát za sebou. Je to bug, se kterým moc neuděláme, pokud se to opakuje, pak se dá zkusit jiný prohlížeč.

**Comment by Yendy** [comment]
S těma klubovnama jsem na to koukal a "období do" je bohužel natvrdo zadáno jako povinné. Změnit se to nedá, protože již existuje obsah, muselo by se to celé předělat. Zadal jsem k poslední klubovně rok 2999, to snad každému dojde :-).

**Comment by kalich5** [comment]
Ok, zkusím to. Jinak všechno teď nahrávám přes Operu, v Chromu x64 to nefunguje. Ale pro prohlížení je v pohodě.

**Comment by kalich5** [comment]
![galerie](https://cloud.githubusercontent.com/assets/8988898/15927130/fe50368e-2e3f-11e6-8792-481fab6af8af.png)

**Comment by kalich5** [comment]
Nastav prosím galerii tak, aby tam nebyly ty 2 prázdná místa

**Comment by Yendy** [comment]
Změněn počet obrázku v galerii na 32 per page.

**Comment by kalich5** [comment]
Mám 2 chyby:
1. V nepřihlášeném stavu jsou vidět odkazy na tábory a galerii. To by podle mě nemělo být
2. Přejmenuj to "HOME" na "Domů" (nebo nějak jinak, ale ať to není anglicky) 
![chyba](https://cloud.githubusercontent.com/assets/8988898/16119103/45740d84-33da-11e6-8ee6-7f48dd52b757.png)

**Comment by kalich5** [comment]
U některých lidí není vidět v profilu jejich oddílové číslo:
Termit ho tam nemá: http://petkari.cz/user/39
Mrak ho tam třeba má: http://petkari.cz/user/43

A přitom oba mají účty propojené

**Comment by Yendy** [comment]
Jo, oddílové číslo tam uživatelé píšou sami, to propojeno není. Je to kvůli upřesnění při registraci, pak už to tak důležité není. Já ho tam automaticky nepřipisuji. Stačí, že je u každého člena.

**Comment by Yendy** [comment]
"Home" jsem tam nechal, protože "Domů" není úplně korektní překlad a mě osobně se nelíbí. Občas se tam píše "Úvod", ale zkusil bych najít něco výstižnějšího.

**Comment by kalich5** [comment]
A co tam třeba napsat třeba "Příspěvky" nebo "Aktuality"

**Comment by kalich5** [comment]
A u těch členů, chybí tam propojení na ty jejich profily. V opačném směru to je, zvážil bych to, že by to číslo byl odkaz na profil člena.

**Comment by Yendy** [comment]
To dočasné zobrazení Tábory, Galerie u nepřihlášeného člena jsem odstranil hned po testování nových pohledů. Asi jsi tam zrovna koukal, když jsem to měnil, teď už to tam není.

**Comment by Yendy** [comment]
A co třeba "Novinky"? Home stránka je řazena od nejnovějšího ...

**Comment by kalich5** [comment]
Jo, novinky to je super nápad :-) dej to tam
